core/src/main/resources/reference.conf

```diff
@@ -1,128 +1,141 @@
+  ### Appmaster JVM argument configuration
+  ###################
+  appmaster {
+    vmargs = "-server -Xms128M -Xmx256M -Xss1M -XX:MaxPermSize=128m -XX:+HeapDumpOnOutOfMemoryError -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=80 -XX:+UseParNewGC -XX:NewRatio=3"
+    extraClasspath = ""
+  }
 
-    ##############################
-    ### Required to change!!
-    ### You need to set the master cluster address here
-    ###
-    ###
-    ### For example, you may start three master
-    ### on node1: bin/master -ip node1 -port 3000
-    ### on node2: bin/master -ip node2 -port 3000
-    ### on node3: bin/master -ip node3 -port 3000
-    ###
-    ### Then you need to set the cluster.masters = ["node1:3000","node2:3000","node3:3000"]
-    cluster {
-      masters = []
-    }
+  ###################
+  ### Executor argument configuration
+  ### Executor JVM can contains multiple tasks
+  ###################
+  executor {
+    vmargs = "-server -Xms128M -Xmx256M -Xss1M -XX:MaxPermSize=128m -XX:+HeapDumpOnOutOfMemoryError -XX:+UseConcMarkSweepGC -XX:CMSInitiatingOccupancyFraction=80 -XX:+UseParNewGC -XX:NewRatio=3"
+    extraClasspath = ""
+  }
 
-    ### Define where the submitted jar file will be stored at
+  ##############################
+  ### Required to change!!
+  ### You need to set the master cluster address here
+  ###
+  ###
+  ### For example, you may start three master
+  ### on node1: bin/master -ip node1 -port 3000
+  ### on node2: bin/master -ip node2 -port 3000
+  ### on node3: bin/master -ip node3 -port 3000
+  ###
+  ### Then you need to set the cluster.masters = ["node1:3000","node2:3000","node3:3000"]
+  cluster {
+    masters = []
+  }
 
-    ### This path follows the hadoop path schema
-    ### For HDFS, use hdfs://host:port/path/
-    ### For FTP, use ftp://host:port/path
-    ### If you want to store on master nodes, then use local directory,
-    ### jarstore.rootpath = "jarstore/" will points to relative directory where master is started.
-    ### jarstore.rootpath = "/jarstore/" will points to absolute directory on master server
-    jarstore.rootpath = "jarstore/"
+  ### Define where the submitted jar file will be stored at
 
-    #########################
-    ### Scheduller for master, it will use this scheduler to schedule resource for
-    ### different applications.
-    ### If you don't know what is this about, don't change it
-    #########################
-    scheduling {
-      scheduler-class = "org.apache.gearpump.cluster.scheduler.PriorityScheduler"
-    }
+  ### This path follows the hadoop path schema
+  ### For HDFS, use hdfs://host:port/path/
+  ### For FTP, use ftp://host:port/path
+  ### If you want to store on master nodes, then use local directory,
+  ### jarstore.rootpath = "jarstore/" will points to relative directory where master is started.
+  ### jarstore.rootpath = "/jarstore/" will points to absolute directory on master server
+  jarstore.rootpath = "jarstore/"
```

Comment:
There is also a section "jarstore.rootpath" in gear.conf
which will override reference.conf.